authorize.py

```diff
@@ -59,7 +59,7 @@ def authorize(event,context):
     #return the value pushed, that is, auth token with expiry time.
     #throw in the email for frontend.
     update_val['auth']['email'] = email
-    ret_val = { "statusCode":200,"isBase64Encoded": False, "headers": { "Content-Type":"application/json" },"body" : json.dumps(update_val)}
```

Comment:
should fix #65

Reply:
Yes. I don't remember when I threw it in, because it would have broken the frontend.